.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Test
```

Comment:
why is this file here? 
It's not present on our main branch https://github.com/workaroundgmbh/bleak/tree/proglove-v0.19.0

Reply:
I got confused during the rebase :D

Reply:
removed it now :)